content/docs/other-guides/usage-reporting.md

```diff
@@ -4,30 +4,39 @@ description = "Opting in or out of anonymous usage data reported from Kubeflow"
 weight = 40
 +++
 
-When enabled, Kubeflow will report **anonymous** usage data using [spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' reporting tool. Spartakus **does not report any personal information**. See [here](https://github.com/kubernetes-incubator/spartakus) for more detail.
-This is entirely voluntary and you can opt out by doing the following:
+When enabled, Kubeflow will report **anonymous** usage data using 
+[Spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' 
+reporting tool. Spartakus does not report any personal information. 
+See the [Spartakus docs](https://github.com/kubernetes-incubator/spartakus) for 
+more detail. 
+
+Allowing usage reporting is entirely voluntary.
+**Reporting usage data is one of the most significant contributions you can make 
+to Kubeflow; so please consider allowing the reporting of usage data.** 
+The data helps the Kubeflow community to improve the project and helps the many 
+companies working on Kubeflow justify continued investment.
+
+**To opt out of usage reporting,** run the following command:
```

Comment:
I might suggest clarifying the instructions as follows

1. If they've already deployed Kubeflow and want to turn of usage reporting then they should run the command

kubectl delete -n ${NAMESPACE} deploy spartakus-volunteer

If the spartakus-volunter deployment doesn't exist then they aren't reporting usage; they could check by running

kubectl get -n ${NAMESPACE} get sparatkus-volunteer

If they want to prevent sparatakus from being deployed then they should follow the instructions below about removing it from app.yaml

Reply:
Done.